docs/help/support-guidelines.md

```diff
@@ -63,11 +63,13 @@ If you can't determine the appropriate repository, ask the maintainers for help
 
 If you want to fix the bug by yourself, discuss the approach with maintainers and submit a pull request.
 
-## Slack
+## Discord
 
-Autoware has a Slack workspace for casual communication between contributors.
+[![Discord](https://img.shields.io/discord/953808765935816715?label=Join%20Autoware%20Discord&style=for-the-badge)](https://discord.gg/Q94UsPvReQ)
```

Comment:
Will you add this badge to https://github.com/autowarefoundation/.github/pull/1 as well? I feel it's good.

Reply:
Done! https://github.com/autowarefoundation/.github/pull/1#discussion_r829801076